examples/phi2/phi2_optimize_template.json

```diff
@@ -96,17 +122,10 @@
             }
         }
     },
-    "pass_flows": [
-        [
-            "convert",
-            "optimize",
-            "perf_tuning"
-        ]
-    ],
     "engine": {
         "evaluate_input_model": true,
         "evaluator": "common_evaluator",
-        "target": "local_system",
+        "target": "local_system_cpu",
```

Comment:
could you add the desired ep like in the llama2 example https://github.com/microsoft/Olive/blob/main/examples/llama2/llama2.py#L106 

Given an EP, Olive can infer the corresponding accelerator. So, I think for the sake of simplicity, we can remove the `target` and `systems` part entirely. The system is local, so it is redundant. Thanks!

Reply:
fixed